src/sys/aio.rs

```diff
@@ -13,38 +13,41 @@ use sys::time::TimeSpec;
 
 /// Mode for `AioCb::fsync`.  Controls whether only data or both data and
 /// metadata are synced.
-#[repr(i32)]
-#[derive(Clone, Copy, Debug, PartialEq)]
-pub enum AioFsyncMode {
-    /// do it like `fsync`
-    O_SYNC = libc::O_SYNC,
-    /// on supported operating systems only, do it like `fdatasync`
-    #[cfg(any(target_os = "openbsd", target_os = "bitrig",
-              target_os = "netbsd", target_os = "macos", target_os = "ios",
-              target_os = "linux"))]
-    O_DSYNC = libc::O_DSYNC
+libc_enum! {
+    #[repr(i32)]
+    pub enum AioFsyncMode {
+        /// do it like `fsync`
+        O_SYNC,
+        /// on supported operating systems only, do it like `fdatasync`
+        #[cfg(any(target_os = "openbsd", target_os = "bitrig",
```

Comment:
Since you're touching these lines I'd ask that you also alphabetize the target OSes and when they're longer than a line and need to be wrapped that you wrap them such that there's 1 OS per line. This applies across this whole PR.